### ISSUE TITLE
Redesign UI

### ISSUE THREAD

**Issue by ac2pic** [CCDirectLink/crosscode-map-editor#25]
The UI can be messy and hard to maintain. 
UI components like the TileSelector is intrusive. It should be a popup window instead of being stuck to the current webpage.

**Comment by Vegita2** [comment]
I agree with you. Showing the tilemap all the time is not the best way to handle that. Something like that is already planned. It should look like the views and toolbars tab from tiled
![image](https://user-images.githubusercontent.com/9483499/57465078-2bee6400-727e-11e9-8f04-7afecc32346f.png)
Basically you select the windows you want to see and your selection is saved for the next time you use the editor. Also there could be shortcuts to toggle views on or off or switch complete workspaces.

**Comment by keanuplayz** [comment]
Ever tried doing something like Paint.NET does?
I have no idea if this works, or is a good idea at all. Just making a suggestion.
![image](https://user-images.githubusercontent.com/50610755/59359188-2cbb5f80-8d2e-11e9-96c2-07f435704ff3.png)

**Comment by ac2pic** [comment]
That's where I would like to see the map editor head towards in terms of the UI.

**Comment by Vegita2** [comment]
The features are not implemented. 
We could split this into multiple issues because this one has quite a few new features
